packages/foreman/gyp/gyp.spec

```diff
@@ -0,0 +1,194 @@
+%global		revision	fcd686f1
+%{expand:	%%global	archivename	gyp-%{version}%{?revision:-git%{revision}}}
+%if 0%{?rhel} && 0%{?rhel} <= 6
+%{!?__python2: %global __python2 /usr/bin/python2}
+%if 0%{?rhel} == 5
+%global __python2 /usr/bin/python26
+%global __os_install_post %__multiple_python_os_install_post
+%endif
+%{!?python2_sitelib: %global python2_sitelib %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
+%{!?python2_sitearch: %global python2_sitearch %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib(1))")}
+%endif
+
+Name:		gyp
+Version:	0.1
+Release:	0.33%{?revision:.%{revision}git}%{?dist}.2
+Summary:	Generate Your Projects
+
+License:	BSD
+URL:		https://gyp.gsrc.io
+# No released tarball avaiable. so the tarball was generated
+# from svn as following:
+#
+# 1. git clone https://chromium.googlesource.com/external/gyp
+# 2. cd gyp
+# 3. version=$(grep version= setup.py|cut -d\' -f2)
+# 4. revision=$(git log --oneline|head -1|cut -d' ' -f1)
+# 5. tar -a --exclude-vcs -cf /tmp/gyp-$version-git$revision.tar.xz *
+Source0:	%{archivename}.tar.xz
+Patch0:		gyp-rpmoptflags.patch
+Patch1:		gyp-ninja-build.patch
+Patch2:		gyp-python3.patch
+Patch3:		gyp-python38.patch
+Patch4:		gyp-fix-cmake.patch
+
+%if 0%{?rhel}
+%if 0%{?rhel} == 5
+BuildRequires:	python26-devel
+%elif 0%{?rhel} < 8
+BuildRequires:	python2-devel
+%else
+BuildRequires:	python3-devel
+%endif
+%if 0%{?rhel} < 8
+BuildRequires:	python2-setuptools
+Requires:	python2-setuptools
+%else
+BuildRequires:	python3-setuptools
+Requires:	python3-setuptools
+%endif
+%else
+BuildRequires:	python3-devel python3-setuptools
+Requires:	python3-setuptools
+%endif
+BuildRequires:	gcc gcc-c++ ninja-build
+BuildArch:	noarch
+
+%description
+GYP is a tool to generates native Visual Studio, Xcode and SCons
+and/or make build files from a platform-independent input format.
+
+Its syntax is a universal cross-platform build representation
+that still allows sufficient per-platform flexibility to accommodate
+irreconcilable differences.
+
+
+%prep
+%autosetup -p1 -c -n %{archivename}
+for i in $(find pylib -name '*.py'); do
+	sed -e '\,#![ \t]*/.*python,{d}' $i > $i.new && touch -r $i $i.new && mv $i.new $i
+done
+
+%build
+%{__python3} setup.py build
+
+
+%install
+%{__python3} setup.py install --root $RPM_BUILD_ROOT --skip-build
+
+
+%check
+%{__python3} gyptest.py test/hello/gyptest-all.py
+
+
+%files
+%license LICENSE
+%doc AUTHORS
+%{_bindir}/gyp
+%{python3_sitelib}/*
+
+
+%changelog
+* Tue mar 24 2020 Zach Huntington-Meath <zhunting@redhat.com> - 0.1-0.33.fcd686f1git.2
+- 
```

Comment:
What are the changes here?

Reply:
Thanks!

Reply:
I decided I wasn't changing it, so I just kept it as is for now. Should I go ahead and bump it just to show we are building it here?

Reply:
If there's no change at all, I'd keep the exact same version as in EPEL including the same release.